src/olympia/addons/tests/test_views.py

```diff
@@ -3015,7 +3015,7 @@ def test_developer_version_serializer_used_for_authors(self):
         # not logged in
         assert 'source' not in self.client.get(self.url).data
 
-        user = UserProfile.objects.create(username='user')
+        user = user_factory()
```

Comment:
drive-by, I replaced all `UserProfile.objects.create(username='user')` in this test file with `user_factory()` for consistency, which amongst other things ensures the user always has an email (which in turn matters for some of the eula/policy tests because of `isSubmissionAllowedFor` permission check).